JavaScript/JavaScriptSDK.Tests/Selenium/Tests.cs

```diff
@@ -23,6 +24,7 @@ public class Tests
         private static IISExpress iisExpress;
         private const string PATH_TO_TESTS = "/Selenium/Tests.html";
         private const string PATH_TO_PERF_TESTS = "/Selenium/PerfTests.html";
+        private static string FirefoxPath = @"C:\Program Files\Mozilla Firefox\firefox.exe";
```

Comment:
I knew you're gonna ask about it 😄. I'll spend more time investigating other options...